packages/core/src/electron-main/theia-electron-window.ts

```diff
@@ -44,6 +44,12 @@ export const TheiaBrowserWindowOptions = Symbol('TheiaBrowserWindowOptions');
 export const WindowApplicationConfig = Symbol('WindowApplicationConfig');
 export type WindowApplicationConfig = FrontendApplicationConfig;
 
+enum ClosingState {
```

Comment:
Suggestion: We should probably export this enum, right?

Reply:
I don't think so: it's an implementation detail that should not leak out of the module.